assets/src/js/helper.js

```diff
@@ -40,3 +40,34 @@ function dokan_get_i18n_date_format( format = true ) {
 
   return datepickerFormat;
 }
+
+/**
+ * Dokan Sweet Alert
+ *
+ * @since 3.2.13
+ *
+ * @param {string} message The event message for notification
+ * @param {object} [options] Configuration of sweet alert modal
+ *
+ * @return {Promise | bool} Return Promise on success, and false on failure
+ */
+ async function dokan_sweet_alert( message = '' , options = {} ) {
```

Comment:
Kindly rename the function name as `dokan_sweetalert` because *sweetalert* us a library name.